.github/workflows/chart_update_on_merge.yml

```diff
@@ -0,0 +1,62 @@
+name: chart-update-on-merge
+
+on:
+  push: # There is no specific "PR merged" event but pushes to protected branch `develop` would only happen during PR merges (and releases) (and Jeff)
+    branches:
+      - develop
+jobs:
+  chart-update:
+    name: Cromwhelm Chart Auto Updater
+    runs-on: self-hosted # Faster machines; see https://github.com/broadinstitute/cromwell/settings/actions/runners
+    steps:
+    - name: Clone Cromwell
+      uses: actions/checkout@v2
+      with:
+        repository: broadinstitute/cromwell
+        token: ${{ secrets.BROADBOT_GITHUB_TOKEN }} # Has to be set at checkout AND later when pushing to work
+        path: cromwell
+    - uses: olafurpg/setup-scala@v10
```

Comment:
Is this an official thing? Do we trust it (now and into the future) to do our builds safely and securely?

Reply:
It is what Denis et al. gave us in the Trivy action, which I took to be an endorsement.